src/model/GameDto.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require('sequelize');
+
+let gameModal = function (sequelize) {
```

Comment:
1. should be "gameModel"
2. your filename is misleading, it says this is a dto but instead you are defining a model here, so I think the filename should be "GameModel.js"